src/main/java/com/spotinst/sdkjava/model/api/ocean/aks/ApiImportClusterLaunchSpecificationAks.java

```diff
@@ -0,0 +1,192 @@
+package com.spotinst.sdkjava.model.api.ocean.aks;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+import com.spotinst.sdkjava.model.bl.ocean.aks.*;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiImportClusterLaunchSpecificationAks implements IPartialUpdateEntity {
```

Comment:
Can we update the class-name to reflect that it is a response object.

Reply:
updated